src/core.lisp

```diff
@@ -18,15 +18,19 @@
   (:import-from #:weblocks/html
                 #:with-html-string)
   (:export
-   #:widget
+   #:ui-widget
    #:*foundation-dependencies*))
 (in-package weblocks-ui/core)
 
 
-(defwidget widget ()
+(defwidget ui-widget ()
   ()
   (:documentation "Use this class as a parent for all widgets, who use UI."))
 
+(defwidget widget (ui-widget)
+  ()
+  (:documentation "'widget' was renamed to 'ui-widget'."))
```

Comment:
Please, add a `initialize-instance :before` specialized on this class. It should do:

```
(warn "Please, switch to the ui-widget class, because `widget' was renamed to `ui-widget' and will be removed after 2020-06-01.")
```

Also it will be nice to add to the code a comment like:

```
TODO: remove after 2020-06-01
```

Also it will be us